tests/test_check_obsfiles_ert_yaml.py

```diff
@@ -0,0 +1,259 @@
+
+    return ert_obs
+
+
+def read_yaml_obs(yaml_obs_file_name: pathlib.Path) -> dict:
+    """This function reads the content of a YAML observation file and returns the information in a dictionary.
+    Args:
+        yaml_obs_file_name: path to the YAML observation file
+    Returns:
+        dictionary that contains the information in a YAML observation file
+    """
+    assert os.path.exists(yaml_obs_file_name) == 1
+    a_yaml_file = open(yaml_obs_file_name, "r")
+    yaml.allow_duplicate_keys = True
+
+    return yaml.load(a_yaml_file, Loader=yaml.FullLoader)
+
+
+def compare(ert_obs_dict: dict, yaml_obs_dict: dict) -> bool:
+    """This function compares if the given dictionaries: ert_obs_dict and yaml_obs_dict contain the same information.
+    Args:
+        ert_obs_dict: dictionary that contains the information in a ERT observation file
+        yaml_obs_dict: dictionary that contains the information in a YAML observation file
+    Returns:
+        True: If the ert_obs_dict and yaml_obs_dict contains the same information
+              Otherwise function stops by assert functions if both dictionaries have diferent information.
+    """
+    yaml_obs = {}
+    for item in yaml_obs_dict:
+        for list_item in yaml_obs_dict[item]:
+            for lost_item in list_item["observations"]:
+                if not list_item["key"] in yaml_obs:
+                    yaml_obs[list_item["key"]] = [[], [], []]
+                yaml_obs[list_item["key"]][0].append(lost_item["date"].toordinal())
+                yaml_obs[list_item["key"]][1].append(float(lost_item["value"]))
+                yaml_obs[list_item["key"]][2].append(float(lost_item["error"]))
+            assert yaml_obs[list_item["key"]][0] == ert_obs_dict[list_item["key"]][0]
+            assert yaml_obs[list_item["key"]][1] == ert_obs_dict[list_item["key"]][1]
+            assert yaml_obs[list_item["key"]][2] == ert_obs_dict[list_item["key"]][2]
+
+    return True
+
+
+def test_check_obsfiles_ert_yaml() -> None:
+    """
+    This function checks if the observation files (complete, training, and test) in ERT and YAML version are equal.
+       Args:
+           None. The file names containig the observation files are harcoded as
+       Returns:
+           None
+    """
+
+###############################################################################################                   
+###############################################################################################                   
+
+    # Load Config
+    config = parse_config(_CONFIG_FILE_NAME, None)
+    
+    #TODO: Create a fake error vector (config.flownet.data_source.simulation.vectors),
+    #      it apear that it needs to be named_dict type.
+    #vectors = configsuite.config.named_dict
+    #print(type(config.flownet.data_source.simulation.vectors))
+    #print(config.flownet.data_source.simulation.vectors)
+    #print(type(vectors))
+    #print(vectors)
```

Comment:
How Could I create a fake config variable? actually, what its needed is config.flownet.data_source.simulation.vectors and this is a name_dict variable, I don't know how to defined and assign values on it. I try to follow what is done in  Config_Parse but It didn't work.

Reply:
It's done in this file: https://github.com/equinor/flownet/blob/master/tests/test_generate_connections.py